src/TideCoreOperation.php

```diff
@@ -233,4 +234,61 @@ public function chagneDiffSettings() {
     }
   }
 
+  /**
+   * Enable TFA.
+   */
+  public static function enableTfaNecessaryModules() {
+    $moduleHandler = \Drupal::service('module_handler');
+    $moduleInstaller = \Drupal::service('module_installer');
+    // Enable Real AES.
+    if (!$moduleHandler->moduleExists('real_aes')) {
+      $moduleInstaller->install(['real_aes']);
+    }
+    // Enable Two-factor Authentication (TFA).
+    if (!$moduleHandler->moduleExists('tfa')) {
+      $moduleInstaller->install(['tfa']);
+    }
+  }
+
+  /**
+   * Disabled site alert for TFA routes.
+   */
+  public static function disabledSiteAlertTfa() {
+    $block = Block::load('tide_site_alert_header');
```

Comment:
You should avoid loading the config here because `tide_core` is not aware of the `tide_alter` module.